test/topology/conftest.py

```diff
@@ -213,13 +212,15 @@ async def manager(request, manager_internal, record_property, mode):
         )
         with open(failed_test_dir_path / f"stacktrace", 'w') as f:
             f.write(report.longreprtext)
-        if request.config.getoption('artifacts_dir_url') is not None:
+        if request.config.getoption('artifacts_dir_url') != "None":
```

Comment:
Does it mean, that the test is executed with `--artifacts_dir_url=None` option?

Reply:
The issue was in another place: in test.py how I pass the parameter to pytest.